scripts/scaletest.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
why is this script in a `scripts/` directory but `coder_init.sh` and `coder_workspacetraffic.sh` are in `terraform/`?